package.json

```diff
@@ -138,7 +138,6 @@
     "react-dnd": "^2.5.1",
     "react-dnd-html5-backend": "^2.5.1",
     "react-dom": "^16.8.1",
-    "react-dropzone": "^4.1.2",
```

Comment:
Something tells me that the yarn lock file should have changes when a dependency is being removed.